File: MODUL-1/SOAL 10/PRAK110-2410817120017-RIKA-FAULIANA-RAHMI.c
#include <stdio.h>
#include <math.h>

int main() {
    // Diketahui
    int alas = 5;
    int tinggi = 12;

    // Menghitung sisi C (hipotenusa)
    double sisiC = sqrt((alas * alas) + (tinggi * tinggi));

    // Menghitung keliling
    double keliling = alas + tinggi + sisiC;

    // Menghitung luas
    double luas = (alas * tinggi) / 2;

    // Output
    printf("Diketahui:\n");
    printf("Alas = %d cm\n", alas);
    printf("Tinggi = %d cm\n\n", tinggi);
    
    printf("Jawab:\n");
    printf("Sisi A = %d cm\n", alas);
    printf("Sisi B = %d cm\n", tinggi);
    printf("Sisi C = %.0f cm\n", sisiC);
    printf("Keliling = %.0f cm\n", keliling);
    printf("Luas = %.0f cm\n", luas);

    return 0;
}

File: MODUL-1/SOAL 10/PRAK110-2410817120017-RIKA-FAULIANA-RAHMI.py
import math

# Diketahui
alas = 5
tinggi = 12

# Menghitung sisi C (hipotenusa)
sisiC = math.sqrt((alas ** 2) + (tinggi ** 2))

# Menghitung keliling
keliling = alas + tinggi + sisiC

# Menghitung luas
luas = (alas * tinggi) / 2

# Output
print("Diketahui:")
print(f"Alas = {alas} cm")
print(f"Tinggi = {tinggi} cm\n")

print("Jawab:")
print(f"Sisi A = {alas} cm")
print(f"Sisi B = {tinggi} cm")
print(f"Sisi C = {int(sisiC)} cm")  # Mengubah ke integer
print(f"Keliling = {int(keliling)} cm")  # Mengubah ke integer
print(f"Luas = {int(luas)} cm")  # Mengubah ke integer
